astropy/io/fits/hdu/table.py

```diff
@@ -1294,7 +1294,7 @@ def _dump_coldefs(self, fileobj):
         for column in self.columns:
             line = [column.name, column.format]
             attrs = ['disp', 'unit', 'dim', 'null', 'bscale', 'bzero']
-            line += ['{:16s}'.format(value if value else '""')
+            line += [f'{value}' if value else '""'
```

Comment:
You should keep the `16s`, so something like this :
```suggestion
            line += ['{!s:16s}'.format(value if value else '""')
```